include/accessor.h

```diff
@@ -186,12 +185,13 @@ class accessor<DataT, Dims, Mode, target::device> : public detail::accessor_base
 		// We currently don't support boundary checking for accessors created using accessor_testspy::make_device_accessor,
 		// which does not set m_oob_indices.
 		if(m_oob_indices != nullptr) {
-			const bool is_within_bounds_lo = all_true(index >= m_accessed_virtual_subrange.offset);
-			const bool is_within_bounds_hi = all_true(index < (m_accessed_virtual_subrange.offset + m_accessed_virtual_subrange.range));
+			const bool is_within_bounds_lo = all_true(index >= m_accessed_buffer_subrange.offset);
```

Comment:
I'm not sure if all this renaming is worth it to touch all this code, since I don't think the new names are any more immediately obvious than the old ones. But I also don't think it's the other way around, so it's fine with me.

Reply:
I'm trying to get rid of the "virtual buffer" vs "backing buffer" terminology because its a remnant from pre-USM Celerity, and these are more easily confused because of "buffer" appearing in both.